doc/index.rst

```diff
@@ -2,15 +2,30 @@ xarray: N-D labeled arrays and datasets in Python
 =================================================
 
 **xarray** (formerly **xray**) is an open source project and Python package
-that aims to bring the labeled data power of pandas_ to the physical sciences,
-by providing N-dimensional variants of the core pandas data structures.
-
-Our goal is to provide a pandas-like and pandas-compatible toolkit for
-analytics on multi-dimensional arrays, rather than the tabular data for which
-pandas excels. Our approach adopts the `Common Data Model`_ for self-
-describing scientific data in widespread use in the Earth sciences:
-``xarray.Dataset`` is an in-memory representation of a netCDF file.
-
+that makes working with labelled multi-dimensional arrays simple,
+efficient, and fun!
+
+Multi-dimensional (a.k.a. N-dimensional, ND) arrays (somtimes called "tensors")
+are an essential part of computational science.
+They are encountered in a wide range of fields, including physics, astronomy,
+geoscience, bioinformatics, engineering, finance, and deep learning.
+In python, numpy_ provides the fundamental data structure and API for
+working with raw ND arrays.
+However, real-world datasets are usually more than just raw numbers;
+they have "labels" which encode information about how the array values map
+to locations in space, time, etc.
+By adopting the the `Common Data Model`_ for self-describing scientific data,
```

Comment:
`the the` -> `the`

Reply:
Maybe (I'm not actually sure htis is better):
`By adopting the self-describing data model of the netCDF file format`